### PR TITLE
docs: add monaco editor guide

### PR DESCRIPTION
**Pull request by betomoedano**
## What kind of change does this pull request introduce?

Introduces a guide to integrate Monaco editor in place of the CodeMirror #327 
## What is the current behavior?

Currently we only have one Guides page

## What is the new behavior?

This PR adds a section called Guides, where we can add more guides in the future

## Checklist

- [X] Documentation
- [X] Tests
- [ ] Ready to be merged
      

Let me know if you like this format or if you have any suggestions, thanks 👍.

### CODE REVIEW

File: website/docs/docs/guides/integrate-monaco-editor.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          onChange={(value) => updateCode(value || "")}
+        />
+      </div>
+    </SandpackStack>
+  );
+}
+```
+
+:::info
+If you are unfamiliar with any of the code in this guide make sure to check the [**Advances Usage**](https://sandpack.codesandbox.io/docs/advanced-usage/provider) section for a better understanding.
```
Comment (danilowoz) [inline website/docs/docs/guides/integrate-monaco-editor.md:82]: Awesome!

File: website/docs/docs/guides/integrate-monaco-editor.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+:::
+
+Finally, using a `SandpackProvider` you can integrate the `MonacoEditor` component like the following example:
+
+```jsx
+export default function MySandpack() {
+  return (
+    <SandpackProvider template="react">
+      <ClasserProvider
```
Comment (danilowoz) [inline website/docs/docs/guides/integrate-monaco-editor.md:91]: I don't think `ClasserProvider` is really necessary for this tutorial, so I'd remove it

File: website/docs/docs/guides/guides-overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-sidebar_position: 7
-title: Guides
+sidebar_position: 1
+title: Guides Overview
 ---
 
 Here you can find some guides or recipes on using the Sandpack in different situations. These examples are basic implementations, and you need to consider improving for production, but you can find more detailed guides in the [advanced usage](/advanced-usage/provider) section.
```
Comment (danilowoz) [inline website/docs/docs/guides/guides-overview.md:6]: Can you add here a link to the page? What do you think?

Comment (betomoedano) [reply website/docs/docs/guides/guides-overview.md:6]: Good idea!

**Pull request merged**